diff --git a/node_modules/@tippy.js/react/dist/tippy-react.esm.js b/node_modules/@tippy.js/react/dist/tippy-react.esm.js
index a2538c4..190f60c 100644
--- a/node_modules/@tippy.js/react/dist/tippy-react.esm.js
+++ b/node_modules/@tippy.js/react/dist/tippy-react.esm.js
@@ -52,10 +52,10 @@ function preserveRef(ref, node) {
 function ssrSafeCreateDiv() {
   return isBrowser && document.createElement('div');
 }
-function updateClassName(tooltip, action, classNames) {
+function updateClassName(box, action, classNames) {
   classNames.split(/\s+/).forEach(function (name) {
     if (name) {
-      tooltip.classList[action](name);
+      box.classList[action](name);
     }
   });
 }
@@ -63,12 +63,12 @@ function updateClassName(tooltip, action, classNames) {
 var useIsomorphicLayoutEffect = isBrowser ? useLayoutEffect : useEffect;
 function useUpdateClassName(component, className, deps) {
   useIsomorphicLayoutEffect(function () {
-    var tooltip = component.instance.popperChildren.tooltip;
+    var box = component.instance.popper.firstElementChild;
 
     if (className) {
-      updateClassName(tooltip, 'add', className);
+      updateClassName(box, 'add', className);
       return function () {
-        updateClassName(tooltip, 'remove', className);
+        updateClassName(box, 'remove', className);
       };
     }
   }, [className].concat(deps));
@@ -128,13 +128,11 @@ function Tippy(_ref) {
       singleton = _ref.singleton,
       _ref$enabled = _ref.enabled,
       enabled = _ref$enabled === void 0 ? true : _ref$enabled,
-      _ref$multiple = _ref.multiple,
-      multiple = _ref$multiple === void 0 ? true : _ref$multiple,
       _ref$ignoreAttributes = _ref.ignoreAttributes,
       ignoreAttributes = _ref$ignoreAttributes === void 0 ? true : _ref$ignoreAttributes,
       __source = _ref.__source,
       __self = _ref.__self,
-      restOfNativeProps = _objectWithoutPropertiesLoose(_ref, ["children", "content", "className", "visible", "singleton", "enabled", "multiple", "ignoreAttributes", "__source", "__self"]);
+      restOfNativeProps = _objectWithoutPropertiesLoose(_ref, ["children", "content", "className", "visible", "singleton", "enabled", "ignoreAttributes", "__source", "__self"]);
 
   var isControlledMode = visible !== undefined;
   var isSingletonMode = singleton !== undefined;
@@ -152,7 +150,6 @@ function Tippy(_ref) {
 
   var props = _extends({
     ignoreAttributes: ignoreAttributes,
-    multiple: multiple
   }, restOfNativeProps, {
     content: component.container
   });
